docs/docs/services/qix-engine/access-control.md

```diff
@@ -84,347 +140,378 @@ Example JWT:
+following additional attributes:
 
-```c
-[resource.resourcetype = "resourcetypevalue"] [OPERATOR] [(((<resource.property = propertyvalue) [OPERATOR (resource.property = propertyvalue)))]
-```
+| Attribute | Description | Example condition |
+| --------- | ----------- | ----------------- |
+| `resource.description` | The object description. | `resource.description = "My custom description"` |
+| `resource._objecttype` | The object type. | `resource._objecttype = "field" or resource.objecttype_ = "my-generic-object"` |
+| `resource.app` | Reference to the app resource that the object is part of. | `resource.app.owner = "john-doe"` |
 
-Argument          | Description
------------------ | -----------
-resource          | Implies that the conditions will be applied to a resource.
-resourcetype      | Implies that the conditions will be applied to a resource of the type defined by the `resourcetypevalue`. You can also use predefined functions for conditions to return property values.
-resourcetypevalue | You must provide at least one resource type value.
-property          | The property name for the resource condition. See Properties for available names.
-propertyvalue     | The value of the selected property name.
-user              | Implies that the conditions will be applied to a user.
+##### The `app` Attribute
 
-The order that you define conditions does not matter. This means that you can define the
-resources first and then the user and/or resource conditions or the other way round.
-However, it is recommended that you are consistent in the order in which you define
-resources and conditions as this simplifies troubleshooting. When using multiple conditions,
-you can group two conditions by wrapping them in parentheses.
+As stated above, if `resource._resourcetype = "App.Object"`, `resource.app` contains the reference to the `"App"`
+resource that the object exists in, and all attributes and [built-in functions](#built-in_function) that apply
+to `"App"` objects are available on `resource.app`.
 
-**Example**
+#### Built-in Functions
 
-```c
-// If the resource is of any type,
-// and the user is a developer,
-// then allow all actions:
-resource.resourcetype = "*" and user.employeeType = "developer" and resource.actions = "*"
-```
+##### `HasPrivilege(<action>)`
 
-## Operators
+Boolean function that returns `true` if the user making the request has already been granted access for the provided
+action. Otherwise returns `false`.
 
-### AND
+**Syntax**
 
-This operator compares two expressions and returns `true` only if both evaluate
-to `true`.
+```py
+resource.HasPrivilege(ACTION)
+```
 
-**Syntax**
+The required parameter `ACTION` shall have a value of any of the supported actions described in [Actions](#Actions).
 
-```c
-(EXPRESSION) && (EXPRESSION)
+**Examples**
 
-// Same as previous, but using "and" notation instead of "&&":
-(EXPRESSION) and (EXPRESSION)
+```py
```

Comment:
Want to use a language that highlights `and`, `or`, strings and so forth. Seems like `&&` and `||` are rendered strangely. At least on GitHub. Any suggestion?

Reply:
I believe `c` worked OK in mkdocs.